sdk/go/pulumi/types.go

```diff
@@ -551,6 +551,29 @@ func UnsafeUnknownOutput(deps []Resource) Output {
 	return output
 }
 
+// UnsafeAwaitedOutput is an output from a Pulumi function or resource that has been resolved.
+//
+// This is a low level API and should be used with care.
+type UnsafeAwaitedOutput struct {
```

Comment:
Super nitpick (that I don't feel strongly about): Should this be named `UnsafeAwaitOutputResult` so it is always exactly next to the associated function?

Reply:
Yeah, that's a good nitpick.